neptune/experiments.py

```diff
@@ -206,15 +218,36 @@ def append_tags(self, tag, *tags):
         self.append_tag(tag, *tags)
 
     def remove_tag(self, tag):
+        """Removes single tag from experiment.
+
+        Args:
+            tag (:obj:`str`): Tag to be removed
+
+        Example:
+            Assuming that `experiment` is an instance of :class:`~neptune.experiments.Experiment`.
+
+            .. code:: python3
+
+                # assuming experiment has tags: `['tag-1', 'tag-2']`.
+                experiment.remove_tag('tag-1')
+
+        Note:
+            Removing a tag that is not assigned to this experiment is silently ignored.
+        """
```

Comment:
@hubertjaworskineptune here you can see how I corrected your docstrings. In particular added: example, args and note